src/scripts/main.js

```diff
@@ -1,3 +1,63 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const appendRaw = document.querySelector('.append-row');
+const removeRaw = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const max = 10;
+const min = 2;
+
+appendRaw.addEventListener('click', e => {
+  const tr = document.createElement('tr');
+
+  for (let item of tbody.children[0].children) {
```

Comment:
`tbody.children[0].children` -  such expression better move to variable with meaningful variable name for a better understanding of what it is and what it will be `item`